doc/mongodb.md

```diff
@@ -0,0 +1,22 @@
+# MongoDB
+
+[MongoDB](https://www.mongodb.com/) is a popular document database that is available. It comes in a variety of flavors, but the version packaged by NixPkgs is generally the community edition.
+
+Because of the licensing of MongoDB, the Nixpkgs derivation that provides the binaries of this database are generally not built and cached by the public nixpkgs cache. Because of this, the initial launch time can be very slow, the first time you use this service locally. On a laptop this time ran to about 3 hours for the initial compile. After the initial build, the start up should be very fast, as with other services, provided you do not update your flake.lock.
+
+If you are using this for your own development, you should either put in place a [local binary cache](https://nixos.wiki/wiki/Binary_Cache) to match your flake, or be aware that the first time you spin up the service it could possibly take a long time to build.
+
+## Options
+
+This module uses most of the same common options that other database modules in this flake use. Some of the most sommon ones are
+
+* `bind` - the address to bind to. Default: 127.0.0.1
+* `port` - the port to serve on. Default: 27017
+* `user` - the MongoDB user to create. Default: default
+* `password` - the password for the MongoDB user. Default: password
+* `extraConfig` - arbitrary extra lines to add to the MongoDB config file. Default: empty string
+* `dataDir` - directory to store data files in. See [datadir](https://github.com/juspay/services-flake/blob/main/doc/datadir.md)
```

Comment:
For options, we are planning to generate them in https://github.com/juspay/services-flake/issues/158. You can remove them from here.

Reply:
Ah, that explains why I haven't been able to find them without trolling through the Nix source.